Mathlib/RingTheory/JacobsonIdeal.lean

```diff
@@ -214,6 +225,45 @@ theorem jacobson_mono {I J : Ideal R} : I ≤ J → I.jacobson ≤ J.jacobson :=
   erw [mem_sInf] at hx ⊢
   exact fun K ⟨hK, hK_max⟩ => hx ⟨Trans.trans h hK, hK_max⟩
 
+theorem jacobson_mul_mem_right {I : Ideal R}
+    (mul_mem_right : ∀ {x y}, x ∈ I → x * y ∈ I) :
+    ∀ {x y}, x ∈ I.jacobson → x * y ∈ I.jacobson := by
+  -- Proof generalized from
+  -- https://ysharifi.wordpress.com/2022/08/16/the-jacobson-radical-definition-and-basic-results/
+  intro x r xJ
+  apply mem_sInf.mpr
+  intro 𝔪 𝔪_mem
+  by_cases r𝔪 : r ∈ 𝔪
+  · apply 𝔪.smul_mem _ r𝔪
+  let 𝔪₀ : Ideal R := {
+    carrier := { a : R | a*r ∈ 𝔪 }
```

Comment:
Is there a mathlib name for this construction?

Reply:
This looks like it's the `Submodule.comap` by `DistribMulAction.toLinearMap` applied to `MulOpposite R`.

Reply:
Thanks! Indeed this worked. I'm not sure it's a change for the better, but it does make the proof shorter. I left a comment explaining what the set actually is.